doc/examples/datasets_train/datasets_train.py

```diff
@@ -30,6 +34,15 @@
 from torch.nn.parallel import DistributedDataParallel
 
 
+###############################################################################
+# MLflow Setup
+# ------------
+#
+# First, let's set up some boilerplate code that will allow us to track and save our
+# ML model to MLflow. This will likely be upstreamed into Ray Train soon, so you won't have
+# to define this yourself anymore. 
```

Comment:
This is now available in master!

Related diff: https://github.com/ray-project/ray/pull/20802/files#diff-882ba864df163bd1bcbb0089df58d319c0358a2344cf654f5d73351decc1d519

Reply:
Awesome! addressed